src/main.ts

```diff
@@ -29,11 +30,18 @@ switch (process.env.NODE_ENV) {
         logger.add(Logger.defaultFileTransport(config.logging.folder));
         if (config.logging.console)
             logger.add(Logger.defaultConsoleTransport());
+        if(config.logging.database)
+        {
+           Logger.BuildDbConnection(config.logging.mongodb_url).then((connection)=>{
+                 if(connection)
+                 logger.add(Logger.defaultDatabaseTransport(config.logging.mongodb_url));  
+           })
+        }  
```

Comment:
(node:7250) [MONGODB DRIVER] Warning: Current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to the MongoClient constructor.